src/testFixtures/java/org/opensearch/neuralsearch/BaseNeuralSearchIT.java

```diff
@@ -208,6 +212,21 @@ protected String prepareModel() {
         return modelId;
     }
 
+    /**
+     * Upload default model and load into the cluster
+     *
+     * @return modelID
+     */
+    @SneakyThrows
+    protected String prepareSparseEncodingModel() {
```

Comment:
No `SneakyThrows` for internal method
```suggestion
    protected String prepareSparseEncodingModel() throws Exception{
```